packages/makeup-hoist-element/src/index.js

```diff
@@ -0,0 +1,79 @@
+let modalEl;
+let modalPlaceholder;
+let containerDiv;
+let prevAriaHiddenSetting;
+let bodyIndexes = [];
+
+function unhoist() {
+    if (modalEl) {
+        if (containerDiv) {
+            const childList = Array.from(containerDiv.childNodes);
+            childList.forEach((child) => {
+                if (child.src === undefined) {
```

Comment:
Can you just do `if (!child.src)`? I assume  `src=""` would also pass but I think it's fine (I don't like `=== undefined` personally, it can lead to bugs if something is set as null)